# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/promiseworker/PromiseWorker.jsm
# Commit: 5ea572e91652
# Full Hash: 5ea572e916522c6a1a96daef9f37427ab245ed57
# Author: Emma Malysz <emalysz@mozilla.com>
# Date: 2021-05-11 02:09:31
# Regressor Bug: 1649602
# File Overlap Count: 2
# Description:
#   Bug 1649602, remove OS.File from PageThumbs.jsm r=Gijs,barret
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101066
# ==============================================================================

diff -r a00207a14a8b -r 5ea572e91652 toolkit/components/promiseworker/PromiseWorker.jsm
--- a/toolkit/components/promiseworker/PromiseWorker.jsm	Mon May 10 16:06:34 2021 +0000
+++ b/toolkit/components/promiseworker/PromiseWorker.jsm	Mon May 10 16:18:44 2021 +0000
@@ -102,6 +102,10 @@
     result.stack = error.stack;
     return result;
   },
+  DOMException(error) {
+    let result = new DOMException(error.message, error.name);
+    return result;
+  },
 };
 
 /**